spec/service/cloud_watch_service_spec.rb

```diff
@@ -0,0 +1,71 @@
+require "rails_helper"
+
+describe CloudWatchService do
+  describe "#week_submissions" do
+    let(:forms_env) { "test" }
+    let(:form_id) { 3 }
+    let(:datapoints) { [ { sum: } ] }
+    let(:sum) { 3.0 }
+
+    around do |example|
+      Timecop.freeze(Time.zone.local(2021, 1, 1, 4, 30, 0)) do
+        example.run
+      end
+    end
+
+    before do
+      allow(Settings).to receive(:forms_env).and_return(forms_env)
+    end
+
+    it "calls the cloudwatch client with get_metric_statistics" do
+      cloudwatch_client = Aws::CloudWatch::Client.new(stub_responses: true)
+
```

Comment:
Nitpick: can we clump together the setup stuff without line breaks before the `expect` pls?